libs/wingsdk/src/target-tf-aws/counter.ts

```diff
@@ -18,7 +19,7 @@ export class Counter extends cloud.CounterBase {
     super(scope, id, props);
 
     this.table = new DynamodbTable(this, "Default", {
-      name: `wingsdk-counter-${this.node.addr}`,
+      name: this.sanitizeName(`${this.node.id}-${this.node.addr}`),
```

Comment:
This still appears to be a generated name of the format: `wingsdk-counter-nodeid-nodeaddr`
You can tell it is a counter now, but not which one.  This may be fine for the generated names, but I'd like a property that I can specify like `name` that will set it to what I've named it as is and maybe a `namePrefix` that I can set if I want a generated name that isn't the default: `namePrefixValue-nodeid-nodeaddr`

Reply:
`node.id` can be defined

if you create a `Counter` with this command `new cloudCounter() as "somethink";`

On AWS Console you will see a DynamoDB `wingsdk-counter-somethink-<addr>`